modules/marfeelAnalyticsAdapter.js

```diff
@@ -20,6 +31,10 @@ function auctionEnd(auction, {timedOut}) {
   window.mrfpb && window.mrfpb.trackAuction(auction)
 }
 
+function registerCMPState(state) {
+  window.mrfpb && window.mrfpb.setCMPState(state);
```

Comment:
it is possible that setCMPState is not present because cache problems, lets add a check here